README.md

```diff
@@ -248,11 +250,84 @@ functionalities, etc.
 
 An example is worth a thousand words. Check out our Hello World demo (available
 to you in different flavors):
-- [Hello World (AMD + bower)](examples/amd-bower/).
-- [Hello World (Node.js + npm)](examples/node-npm/).
-- [Hello World (plain JavaScript)](examples/plain-javascript/).
+- [Hello World (AMD + bower)](examples/hello-world-amd-bower/).
+- [Hello World (Node.js + npm)](examples/hello-world-node-npm/).
+- [Hello World (plain JavaScript)](examples/hello-world-plain-javascript/).
 
 
+### Performance
+
+When formatting or parsing, there's actually a two step process: (a) the
+formatter (or parser) creation and (b) its execution, where creation takes
+considerably more time (more expensive) than execution. The difference is an
+order of magnitude. In the creation phase, Globalize traverses the CLDR tree,
+processes data (e.g., expands date patterns, parses plural rules, etc), and
+returns a function that actually executes the formatting or parsing.
+
+```js
+// Formatter creation.
+var formatter = Globalize.numberFormatter();
+
+// Formatter execution (roughly 10x faster than above).
+formatter( Math.PI );
+// > 3.141
+```
+
+As a rule of thumb for optimal performance, cache your formatters and parsers.
+For example: (a) on iterations, generate them outside the loop and reuse while
+looping; (b) on server applications, generate them in advance and execute when
+requests arrive.
+
+### Compilation and the Runtime modules
+
+You really should take advantage of compiling your formatters and/or parsers
+during build time when deploying to production. It's much faster than generating
+them in real-time and it's also much smaller (i.e., better loading
+performance).
+
+To illustrate, a Globalize application looks like:
+
+```html
+<script src="cldrjs/cldr.js"></script>
+<script src="cldrjs/cldr/...js"></script>
+<script src="globalize.js"></script>
+<script src="globalize/number.js"></script>
+<script src="globalize/...js"></script>
+<script>
+// Feed Globalize on CLDR data and on translation messages.
+Globalize.load({
+  ... CLDR data ...
+});
+
+// Then, use Globalize formatters and parsers.
+</script>
+```
+
+A precompiled Globalize application using the runtime modules looks like:
+
+```html
+<script src="globalize-runtime.js"></script>
+<script src="globalize-runtime/number.js"></script>
+<script src="globalize-runtime/...js"></script>
+<script src="my-compiled-formatters-and-parsers.js"></script>
+<script>
+// Use Globalize formatters and parsers.
+</script>
+```
+
+Your compiled formatters and parsers allow you to skip a big part of the library
+and also allow you to skip loading CLDR data. Because, they have already been
```

Comment:
Combine the two sentences: 
"Your compiled formatters and parsers allow you to skip a big part of the library and skip loading CLDR data because they…"